examples/sample_ssh_config

```diff
@@ -0,0 +1,49 @@
+## For ALL ##
+Host *
+        KexAlgorithms +diffie-hellman-group1-sha1
+        Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
+
+## For NAS Updated 20190315 ##
+Host pfe pfe-last pfe.nas.nasa.gov pfe-last.nas.nasa.gov
+        ProxyCommand ssh shzimmer@sfe2.nas.nasa.gov /usr/local/bin/ssh-balance %h
```

Comment:
I think you should remove your username and replace it with something like <NAS USER NAME>